Cargo.toml

```diff
@@ -6,3 +6,12 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+clap = "4.3.17"
+derive_builder = "0.12.0"
+regex = "1.9.1"
+tree-sitter = "0.20.10"
+tree-sitter-grep = { git = "https://github.com/helixbass/tree-sitter-grep", rev = "3d4682c" }
+tree-sitter-rust = "0.20.3"
```

Comment:
It seems like it might be a little silly to have this crate depend on all of these also (along with `tree-sitter-grep` depending on them)?

Not sure what that would entail to eg "delegate" that dependency, maybe some exposed API from `tree-sitter-grep` to get the `tree_sitter::Language` for a given `SupportedLanguage` or something like that?

Reply:
Yeah, or just a straight re-export of `tree-sitter-rust` from somewhere in `tree-sitter-grep`. (Maybe feature flagged.)

Reply:
Ya actually I ended up doing some crate re-exporting in a recent PR (for rule-author "consumers" I guess)

Here what seems to have been the pattern is that I exposed `SupportedLanguage` on `tree_sitter_grep` and `tree-sitter-lint` is just re-using that as its own "language" type, and a `SupportedLanguage` can give you its corresponding `tree_sitter::Language`